sprint-implementation/outline.md

```diff
@@ -0,0 +1,76 @@
+# Implementig in Sprints Learning Path Outline
+
+## Learning Goals
+
+- The value of this sprint guide
+- Understanding prerequisites to successful InnerSource sprint
+- Learn additional activities for Sprint Planning
+- What to do during the sprint
+- What to do after the sprint
+
+## Target Audience
+
+- Teams who engage into InnerSource work, both Trusted Committers and Contributors
+- People who work/interface with these teams 
+    - Product Owners
+    - Scrum masters
+    - Project management
+    - Team leaders
+
+## Outline
+
+### Introduction
+
+- Why do we need this guide
+
+### Project prerequisites
+- InnerSource checklist
+    - Establish communication channel
+    - README.md
+        - Value of README.md
+        - Structure of README.md
+        - Examples of great README.md files
+    - CONTRIBUTING.md
+        - Value of CONTRIBUTING.md
+        - Structure of CONTRIBUTING.md
+        - Examples of great CONTRIBUTING.md files
+
+### Contribution prerequisites
+- General contribution guidelines
+    - Establish rules for your team
+        - Decide who does PR reviews and how long can a contributor wait for the review
+        - Decide who supports contributors (see next section)
+        - Decide when you need a design review prior to accepting pull requests (Request for Change practice)
+    - Establish rules for contributors and spell them out in CONTRIBUTING.md - see prior section
+- How to organize your backlog
+    - When contribution is more than one story
+        - How to deal with large changes requested by the Contributor's team
+        - What to do if their change has downstream effect
+    - The importance of acceptance criteria
+    - Indications of Contributor's commitment
+    - Managing dependencies
+
+### Spring Planning
+- Contributor's point of contact
+    - Their responsibilities
+    - Communicating with Contributor
+
+### During the Sprint
+- Ensure that Contributor's supporter is able to prioritize the support work over team backlog
+- Helpful tips for PR reviews
+    - Remove whitespace discussions from your code reviews - use linters
```

Comment:
linters and automated tests could also go in pre-requesites.

Reply:
I want to separate the suggestions from the true prerequisites. In this case, collaboration is still possible without automated linters, but they will make life oh-so-much-easier. Which is why I put these into tips category.
Do you still think they should be moved into prerequisites, or is this sufficient? We can link into this section from prerequisites article, too.

Reply:
I like splitting out minimal info from deep dive for pre-requisites (and all advice we give).

Reply:
done